app/models/tslr_claim.rb

```diff
@@ -37,8 +40,11 @@ class TslrClaim < ApplicationRecord
   validates :address_line_3,    on: :address, presence: {message: "Enter your town or city"}
   validates :postcode,          on: :address, presence: {message: "Enter your postcode"}
   validates :date_of_birth,     on: :"date-of-birth", presence: {message: "Enter your date of birth"}
+  validates :teacher_reference_number, on: :"teacher-reference-number", presence: {message: "Enter your teacher reference number"}
+  validate :normalised_trn_length, on: :"teacher-reference-number"
```

Comment:
This has me wondering if we _always_ want to validate the format of the TRN, rather than just in the `"teacher-reference-number"` scope. i.e. what happens if it gets changed elsewhere for whatever reason? would the system be safer if we validate the format of the TRN as long as it isn't `nil`?

Reply:
Great call, we talk this through and think this is the right approach.